owasp-suppressions.xml

```diff
@@ -39,6 +39,14 @@
         <packageUrl regex="true">^pkg:maven/com\.squareup\.wire/wire\-schema\-jvm@.*$</packageUrl>
         <cpe>cpe:/a:wire:wire</cpe>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: protobuf-java-util-3.19.4.jar
+   mismatching with kafka-protobuf-serializer
```

Comment:
this package and this cpe seem to match (the comment mentions kafka which doesn't appear in the actual suppression data). If we're using a protobuf that old, it likely _does_ have vulns and should be updated (try 3.21.7)

Reply:
This is the dependency check result that I am getting without this suppression:
<img width="1728" alt="image" src="https://github.com/hypertrace/entity-service/assets/97032782/d68465fb-a71e-408d-a897-fdb38caefd74">
According to this the `protobuf-java-util-3.19.4.jar` dependency is coming from `kafka-bom (kafka-protobuf-serializer )` itself. What should we do here?

Reply:
ok so it's a real issue, and the vulnerable lib is a dependency of kafka. We can either update the version here, or update it in our bom and then include the updated bom here. If it's not used other than via kafka (i.e. not directly), I'd say we should do the bom else the local constraint. We often use this lib, so it's probably already in the repo.

Reply:
https://github.com/hypertrace/kafka-streams-framework/pull/64